testData/completion/references/ForLoopVariable3.hx

```diff
@@ -0,0 +1,10 @@
+class ForLoopVariable1 {
+  public static function main() {
+    var items = [1, 2, 3, 4, 5];
```

Comment:
If you add another variable that isn't used in the for loops at all, will it still work?  Say, at this point, you add:
``` var inept:String = "blah"```